src/app/[locale]/history/history-client.tsx

```diff
@@ -84,7 +86,7 @@ export default function HistoryView() {
                       style={{ wordBreak: 'break-word', overflowWrap: 'break-word' }}
                     >
                       <Link
-                        href="/rest-client"
+                        href={item.appRouterURL}
                         className={`link link-font_size_small ${themeValue === 'dark' ? 'link--dark' : 'link--light'}`}
                       >
```

Comment:
```suggestion
                        className={`link link-font_size_small ${themeValue === 'dark' ? 'link--dark' : 'link--light'}`}
                        onClick={(e) => e.stopPropagation()}
                      >
```

Add stopPropagation on the Link please, so that clicking the link does not open the data section.